draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1418,8 +1421,7 @@ Centralized Registry:
   registries are outside the scope of this specification and is NOT RECOMMENDED.
 
 Distributed applications generating UUIDs at a variety of hosts MUST
-be willing to rely on the random number source at all hosts.  If this
-is not feasible, the namespace variant should be used.
+be willing to rely on the random number source at all hosts.
```

Comment:
Do we really want to suggest to people that they use the namespace variant?  If a system has let's say a known insecure random number generator, then sure that is going to affect the quality of the output, but I'm not sure that's a strong enough concern to suggest the namespace variant.

Reply:
This was text from 4122: https://www.rfc-editor.org/rfc/rfc4122#section-6

If we want to have a larger discussion about this text I am fine with opening an issue and shooting something to the mailer.